Tools/MSBuildTaskExtractChangelog/MSBuildTaskExtractChangelog.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Framework" Version="16.10.0" />
+    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="16.10.0" />
+  </ItemGroup>
+
+  <Target Name="CopyDllToProjDir" AfterTargets="AfterBuild">
```

Comment:
This is simply done so that it is easier to read when imported (see line 3 of Realm/Realm/Realm.csproj)